### PR TITLE
Sunset SDK (Player)

### PR DESCRIPTION
**Pull request by johansatge**
Sunset the SDK: stop loading a player and disable public API.

### CODE REVIEW

**Review by Supergallus — APPROVED** [review]
🧹 🚢

**Review by Cycymomo — APPROVED** [review]

File: src/core/player.js [hunk 1]
```diff
@@ -35,6 +35,33 @@ DM.provide('',
         if (!options || typeof options !== 'object')
             throw new Error("Missing 'options' parameter for DM.player()");
 
+        console.warn('DEPRECATED: Legacy JS SDK integration method is deprecated. Please consider using Player Embeds https://faq.dailymotion.com/hc/en-us/articles/4411096679954-Integrate-your-Player-Embed');
```
Comment (Cycymomo) [inline src/core/player.js:38]: I would even put the warn before the 2 `throw new Error` ?

no matter the error, we want the warn to be displayed all the time ?

Comment (johansatge) [reply src/core/player.js]: 👍 done

**Review by Cycymomo — APPROVED** [review]
✨

**Pull request merged**